src/include/sof/audio/module_adapter/library/module_api_ver.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ *
+ * Author: Pawel Dobrowolski <pawelx.dobrowolski@intel.com>
+ */
+
+#ifndef __MODULE_API_VER_H__
+#define __MODULE_API_VER_H__
+
+/*
+ *  Api version 5.0.0 for sof loadable modules
+ */
+
+#define _MAJOR_API_MODULE_VERSION 5
+#define _MIDDLE_API_MODULE_VERSION 0
+#define _MINOR_API_MODULE_VERSION 0
```

Comment:
We should align the prefix with the structures i.e. ```SOF_MODULE_API_VERSION_MAJOR``` that way all module API macros and APIS will start with sof_module_api_ or similar i.e. it obvious to the developer using the API .

Reply:
@pjdobrowolski This is still not addressed, right? Please add a comment if you think this should not be changed, otherwise it's impossible to track the PR status.